src-java/northbound-service/northbound-api/src/main/java/org/openkilda/northbound/dto/v2/haflows/HaFlowValidationResult.java

```diff
@@ -0,0 +1,36 @@
+/* Copyright 2023 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.northbound.dto.v2.haflows;
+
+import org.openkilda.northbound.dto.v1.flows.FlowValidationDto;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@Builder
+@AllArgsConstructor
+@JsonNaming(SnakeCaseStrategy.class)
+public class HaFlowValidationResult {
+    Boolean asExpected;
```

Comment:
this might look like a typo. Maybe some other name? What does it mean when asExpected is null?

Reply:
fixed